app/components/organisms/regulars-hero/regulars-hero.styles.tsx

```diff
@@ -0,0 +1,93 @@
+import styled, {css} from 'styled-components';
+
+import {select} from '../../../utils';
+import {Paragraph} from '../../atoms/paragraph';
+
+const desktopMediaQuery = css`
+  ${select(({mediaQueries}) => mediaQueries.desktop)}
+`;
+
+export const RegularsHeroContainer = styled.header`
+  display: flex;
+  max-width: ${select(({sizes}) => sizes.maxWidth)};
+  margin: 1rem auto 0;
+  flex-direction: column;
+
+  ${desktopMediaQuery} {
+    flex-direction: row;
+  }
+`;
+
+export const PrimaryContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+
+  background-color: transparent;
+  border-bottom: ${select(({sizes}) => sizes.borderWidth)} solid
+    ${select(({colors}) => colors.black)};
+
+  ${desktopMediaQuery} {
+    flex: 1.3;
+  }
+`;
+
+export const Description = styled(Paragraph)`
+  display: block;
+
+  background-color: ${select(({colors}) => colors.black)};
+  color: ${select(({colors}) => colors.background)};
+
+  padding: 1.5rem;
+
+  ${desktopMediaQuery} {
+    padding: 1.5rem 5.5rem;
+  }
+`;
+
+export const SecondaryContainer = styled.div`
+  position: relative;
+  border-bottom: 4px solid ${select(({colors}) => colors.black)};
+
+  ${desktopMediaQuery} {
+    flex: 1;
+    border: 4px solid ${select(({colors}) => colors.black)};
+  }
+`;
```

Comment:
in the main page i believe we use `3px` for the border's `width`. if there's a reason to use `4px` let me know!

Reply:
That should actually be using the `borderWidth` prop from the theme, thanks for pointing it out!